File: Java/#832_flipAndInvertImage.java
class Solution {
    public int[][] flipAndInvertImage(int[][] A) {
        for(int i=0;i<A.length;i++){
            for(int j=0, n = A[i].length-1;j<=n;j++,n--){
                int t = A[i][j]^1;
                A[i][j] = A[i][n]^1;
                A[i][n]=t;
                
            }
        }
        return A;
    }
}

File: Python/#832_flipAndInvertImage.py
class Solution:
    def flipAndInvertImage(self, A: List[List[int]]) -> List[List[int]]:
        res = []
        for each in A:
            each.reverse()
            res.append(each)
        for i in res:
            for j in range(len(i)):
                i[j] = i[j]^1
        return res
        
